modules/claims_api/app/services/claims_api/disability_compensation/pdf_generation_service.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'claims_api/v2/disability_compensation_pdf_mapper'
+require 'pdf_generator_service/pdf_client'
+
+module ClaimsApi
+  module DisabilityCompensation
+    class PdfGenerationService < ClaimsApi::Service
+      EVSS_DOCUMENT_TYPE = 'L023'
+
+      def generate(claim_id, middle_initial) # rubocop:disable Metrics/MethodLength
+        log_service_progress(claim_id, 'pdf',
+                             "526EZ PDF generator started for claim #{claim_id}")
+
+        auto_claim = get_claim(claim_id)
+
+        # Reset for a rerun on this
+        set_pending_state_on_claim(auto_claim) unless auto_claim.status == pending_state_value
```

Comment:
this [removes the mocking](https://github.com/department-of-veterans-affairs/vets-api/blob/e89b727d9093f66fa10ed370c67946c009768e3d/modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_pdf_generator.rb#L18) we just added

Reply:
we actually can skip that entire job in the synchronous flow, so we could just ignore calling the pdf service in the controller in sandbox